sdk/src/main/java/com/bugsnag/android/JsonStream.java

```diff
@@ -53,9 +56,10 @@ public void value(@NonNull File file) throws IOException {
         beforeValue(false); // add comma if in array
 
         // Copy the file contents onto the stream
-        FileReader input = null;
+        Reader input = null;
         try {
-            input = new FileReader(file);
+            FileInputStream fis = new FileInputStream(file);
+            input = new BufferedReader(new InputStreamReader(fis, "UTF-8"));
```

Comment:
do any of these objects also need closing?

Reply:
I believe calling `BufferedReader#close` also closes the `InputStreamReader`, which in turn closes the `FileInputStream`, which means that everything is automatically closed.